zipline/data/data_portal.py

```diff
@@ -1728,21 +1729,29 @@ def _get_minute_count_for_transform(self, ending_minute, days_count):
         # we get all the minutes for the last (bars - 1) days, then add
         # all the minutes so far today.  the +2 is to account for ignoring
         # today, and the previous day, in doing the math.
-        previous_day = \
-            self.trading_schedule.previous_execution_day(ending_minute)
-        days = self.trading_schedule.execution_days_in_range(
-            self.trading_schedule.add_execution_days(-days_count + 2,
-                                                     previous_day),
-            previous_day,
+        session_for_minute = self.trading_calendar.minute_to_session_label(
```

Comment:
Just naming-wise, maybe `session_for_ending_minute`?  Does this roll forward or backward for non-market minutes?